README.md

```diff
@@ -0,0 +1,5 @@
+# Media Consent
+
+[![CircleCI](https://circleci.com/gh/srobo/media-consent.svg?style=svg)](https://circleci.com/gh/srobo/media-consent)
+
+PDFs can be downloaded from the 'Artifacts' tab in CircleCI.
```

Comment:
This tab only shows up if you're logged in. Is this expected? Should we note this?
For this repo I think it's fine if the artefacts are public, though I don't think it's a big deal (a comment here would suffice for now IMO).